src/commands/transaction/sign.js

```diff
@@ -62,6 +57,11 @@ export default class SignCommand extends BaseCommand {
 					},
 		});
 
+		const transactionInput =
```

Comment:
Is it necessary to change the order?
If this is necessary, prefer to change order consistently other commands too.

Reply:
@mitsujutsu In general we don't want to prompt a user for their sensitive inputs if we already know the other inputs are invalid.